web/src/main/java/com/bazaarvoice/emodb/web/settings/DatabusSuppressedEventConditionAdminTask.java

```diff
@@ -0,0 +1,42 @@
+package com.bazaarvoice.emodb.web.settings;
+
+import com.bazaarvoice.emodb.common.dropwizard.task.TaskRegistry;
+import com.bazaarvoice.emodb.databus.SuppressedEventCondition;
+import com.bazaarvoice.emodb.sor.condition.Condition;
+import com.bazaarvoice.emodb.sor.condition.Conditions;
+import com.bazaarvoice.emodb.sor.delta.deser.ParseException;
+import com.google.inject.Inject;
+
+/**
+ * Task for managing the condition used to suppress databus events when a subscription opts to ignore suppressed events.
+ *
+ * Examples:
+ *
+ * View the current condition:
+ * <code>
+ *     $ curl -XPOST "localhost:8081/tasks/databus-suppressed-event-condition"
+ * </code>
+ *
+ * Update the condition condition:
+ * <code>
+ *     $ curl -XPOST 'localhost:8081/tasks/databus-suppressed-event-condition?value=\{..,"~tags":contains("re-etl")\}'
```

Comment:
Based on previous comment, it may be confusing to add an "exclusion" condition without adding a "not" to the provided condition.